sources/include/parser/parserutils.hpp

```diff
@@ -46,6 +37,57 @@ namespace citygml {
         return TransformationMatrix(matrix);
     }
 
+    template<class T> inline T parseNumber(const char* str, char** end);
+
+    template<> inline float parseNumber<float>(const char* str, char** end) {
+        return std::strtof(str, end);
+    }
+
+    template<> inline double parseNumber<double>(const char* str, char** end) {
+        return std::strtod(str, end);
+    }
+
+    template<typename T,
+            std::enable_if_t<std::is_floating_point_v<T>, bool> = true>
```

Comment:
`enable_if` is technically not necessary, as T is only instantiated for float or double at the moment.

To get this compiled with C++11 (and GCC) you'll need:
```suggestion
    template<typename T,
            typename std::enable_if<std::is_floating_point<T>::value, bool>::type = true>
```

Reply:
Thanks for the pointer. I'll remove it. It's not necessary.